content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

```diff
@@ -442,13 +442,13 @@ behavior:
       periodSeconds: 15
     selectPolicy: Max
 ```
-For scaling down the stabilization window is _300_ seconds(or the value of the
+For scaling down the stabilization window is _300_ seconds (or the value of the
 `--horizontal-pod-autoscaler-downscale-stabilization` flag if provided). There is only a single policy
 for scaling down which allows a 100% of the currently running replicas to be removed which
 means the scaling target can be scaled down to the minimum allowed replicas.
 For scaling up there is no stabilization window. When the metrics indicate that the target should be
 scaled up the target is scaled up immediately. There are 2 policies where 4 pods or a 100% of the currently
-running replicas will be added every 15 seconds till the HPA reaches its steady state.
+running replicas will be added every 15 seconds until the HPA reaches its steady state.
```

Comment:
"till" sounds better than "until" ?

Reply:
I'm sorry, I mistook this for an informal spelling of "until", I wasn't aware that "till" is actually a standalone word in English.
https://www.merriam-webster.com/dictionary/till
TIL!